### ISSUE TITLE
Atan2 from numpy not implemented

### ISSUE THREAD

**Issue by EmilyBourne** [pyccel/pyccel#204]
The following code  : 
```
from pyccel.decorators import external

@external
def test_arctan2():
    from numpy import arctan2
    return arctan2(2.0)

from pyccel.epyccel import epyccel

test2=epyccel(test_arctan2)
test2()
```

Generates the following error : 
```
pyccel: [semantic] 
 |error: [4,5-23]| Undefined function (arctan2)

Traceback (most recent call last):
  File "/home/admlocal/.local/lib/python3.6/site-packages/pyccel/parser/semantic.py", line 225, in annotate
    ast = self._visit(ast, **settings)
  File "/home/admlocal/.local/lib/python3.6/site-packages/pyccel/parser/semantic.py", line 1056, in _visit
    return getattr(self, annotation_method)(expr, **settings)
  File "/home/admlocal/.local/lib/python3.6/site-packages/pyccel/parser/semantic.py", line 1070, in _visit_Tuple
    ls = [self._visit(i, **settings) for i in expr]
  File "/home/admlocal/.local/lib/python3.6/site-packages/pyccel/parser/semantic.py", line 1070, in <listcomp>
    ls = [self._visit(i, **settings) for i in expr]
  File "/home/admlocal/.local/lib/python3.6/site-packages/pyccel/parser/semantic.py", line 1056, in _visit
    return getattr(self, annotation_method)(expr, **settings)
  File "/home/admlocal/.local/lib/python3.6/site-packages/pyccel/parser/semantic.py", line 2481, in _visit_FunctionDef
    expr.body]
  File "/home/admlocal/.local/lib/python3.6/site-packages/pyccel/parser/semantic.py", line 2480, in <listcomp>
    body = [self._visit(i, **settings) for i in
  File "/home/admlocal/.local/lib/python3.6/site-packages/pyccel/parser/semantic.py", line 1056, in _visit
    return getattr(self, annotation_method)(expr, **settings)
  File "/home/admlocal/.local/lib/python3.6/site-packages/pyccel/parser/semantic.py", line 2354, in _visit_Return
    for assign in assigns]
  File "/home/admlocal/.local/lib/python3.6/site-packages/pyccel/parser/semantic.py", line 2354, in <listcomp>
    for assign in assigns]
  File "/home/admlocal/.local/lib/python3.6/site-packages/pyccel/parser/semantic.py", line 2033, in _visit_Assign
    self.insert_symbolic_function(new_expr)
  File "/home/admlocal/.local/lib/python3.6/site-packages/pyccel/parser/base.py", line 508, in insert_symbolic_function
    raise TypeError('Expected a symbolic_function')
TypeError: Expected a symbolic_function
```

**Comment by EmilyBourne** [comment]
This  can be fixed by adding a function in ast/numpyext.py and adding it to the dictionary math_functions in ast/utilities.py. I'm not sure exactly what is being done with StdFactKB, so I'm not sure how to modify the standard function to get a version with 2 arguments as is needed here

**Comment by yguclu** [comment]
Fixed after merging #206
